### PR TITLE
Feature/instructorUserMangement

### PR DESCRIPTION
**Pull request by Connor-Thorpe01**

### CODE REVIEW

**Review by AndrewDoering01 — APPROVED** [review]

**Review by ryesing — APPROVED** [review]

File: backend/api/handlers/admin/UpdateUserTypeHandler.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		c.JSON(http.StatusBadRequest, gin.H{"error": "Invalid UserID format"})
+		return
+	}
+
+	updateUser.AdminID = adminID
+	dbc := c.MustGet("db").(*gorm.DB)
+	err = db.SetUserType(updateUser, dbc)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err})
+	}
```
Comment (ALackey-ai) [inline backend/api/handlers/admin/UpdateUserTypeHandler.go:43]: Did you want to return a 204 on success here?

File: backend/api/handlers/session/GetSessionByIdHandler.go [hunk 2]
```diff
@@ -1,15 +1,56 @@
...
+		c.JSON(http.StatusOK, gin.H{
+			"sessions": sessions,
+		})
+		return
+	} else {
+		c.JSON(http.StatusNotFound, gin.H{
+			"message": fmt.Sprintf("could not find sessions for userid: %s", userID),
+		})
+		return
+	}
```
Comment (ALackey-ai) [inline backend/api/handlers/session/GetSessionByIdHandler.go:55]: Same here, did you want to return a 204 on success here?

File: backend/api/handlers/admin/UpdateUserTypeHandler.go (hunk 1 above)
Comment (Connor-Thorpe01) [reply backend/api/handlers/admin/UpdateUserTypeHandler.go:43]: yes we do

File: backend/api/handlers/session/GetSessionByIdHandler.go (hunk 2 above)
Comment (Connor-Thorpe01) [reply backend/api/handlers/session/GetSessionByIdHandler.go]: no, here we return sessions above see lines 46 and 36 so we return 200. If we get to this point that means we did not find any sessions for the given user_id and we should return StatusNotFound

Comment (ALackey-ai) [reply backend/api/handlers/session/GetSessionByIdHandler.go]: Cool

**Pull request merged**